easybuild/easyconfigs/q/Qt5/Qt5-5.15.11-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,94 @@
+easyblock = 'EB_Qt'
+
+name = 'Qt5'
+version = '5.15.11'
```

Comment:
We should at least bump this to `5.15.12` (cc @Neves-P)

Reply:
Definitely, will do 👍

Reply:
I've tried bumping the `Qt5` install on my laptop to the latest version that is available as open source (5.15.12) and ran into the same compilation error as https://github.com/easybuilders/easybuild-easyconfigs/pull/19320#issuecomment-1964431589. Going for the patched fork from KDE might be the next viable approach.